packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -35,15 +35,15 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
   private _stickyAboveRef = React.createRef<HTMLDivElement>();
   private _stickyBelowRef = React.createRef<HTMLDivElement>();
   private _contentContainer = React.createRef<HTMLDivElement>();
-  private _subscribers: Set<Function>;
-  private _stickies: Set<Sticky>;
+  private _subscribers: Function[];
```

Comment:
Switched Set to arrays here--more verbose but should hopefully have the same behavior

Reply:
Awesome catch!!!